src/domain/entities/editorTools.ts

```diff
@@ -12,11 +12,6 @@ export default interface EditorTool {
    */
   name: string;
 
-  /**
-   * User-friendly plugin title
-   */
-  title: string;
```

Comment:
why this property has been removed? I guess we need it - it is a human-readable title of the tool displayed in Marketplace and Editor's Toolbox

Reply:
turned back